testsuite/tests/kuadrant/authorino/priority/__init__.py

```diff
@@ -0,0 +1 @@
+"""https://github.com/Kuadrant/authorino/blob/main/docs/features.md#common-feature-priorities"""
```

Comment:
In my opinion, this is a good place to leave a description of the entire package. It is acceptable according to the PEP convention: `A package may be documented in the module docstring of the __init__.py file in the package directory.` (https://peps.python.org/pep-0257). Since other modules will have descriptions for their own content and posting a link to testing features in each module is a bit extra.

Reply:
I got your point and will remove it. Although this file would not be empty, since it would have documentation and everyone would know about it 🤓.